plugins/package-managers/gradle/src/main/kotlin/Gradle.kt

```diff
@@ -290,4 +290,8 @@ class Gradle(
             }
         }
     }
+
+    override fun close() {
+        maven.close()
```

Comment:
is there a clever way in kotlin to do a silent close, i.e. ignore any exceptions that are thrown in the process?

Reply:
See https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/use.html.

Reply:
applied suggestion